synchronizer/connectors/jira.py

```diff
@@ -10,9 +13,20 @@ def __init__(self, **kwargs):
         self.server = kwargs['server']
         self.auth = requests.auth.HTTPBasicAuth(
             kwargs['login'],
-            kwargs['password']
+            self.decrypt(current_app.config['ENCRYPTED_KEY'], kwargs['password'])

```

Comment:
Encryption/Decryption should be applied to password/token field in Connector model. Both operations should be available for ALL connectors. This means that you should move decrypt/encrypt logic outside of connectors (connector should get already decrypted password and doesn't care about decryption at all)